programs/bpf/rust/128bit/Cargo.toml

```diff
@@ -11,12 +11,12 @@ homepage = "https://solana.com/"
 edition = "2018"
 
 [dependencies]
-solana-sdk = { path = "../../../../sdk/", version = "1.5.0", default-features = false }
-solana-bpf-rust-128bit-dep = { path = "../128bit_dep", version = "1.5.0", default-features = false }
+solana-program-sdk = { path = "../../../../sdk/program", version = "1.5.0" }
+solana-bpf-rust-128bit-dep = { path = "../128bit_dep", version = "1.5.0" }
 
 [features]
```

Comment:
😍 

Get rid of the `[features]` entry now that there are no features?

Reply:
Will do